### PR TITLE
[JBTM-3277] force curl to use ipv4 with LRA and tomcat

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.redhat.com/browse/JBTM-3277
https://issues.redhat.com/browse/JBTM-3282

### CODE REVIEW

File: rts/lra-examples/run.sh [hunk 1]
```diff
@@ -72,12 +77,12 @@ function start_and_test_service {
 
   if [ "$1" = "true" ]; then
     echo " ===== waiting for recovery ......."
-    curl http://localhost:${coord_port}/lra-recovery-coordinator/recovery 
+    curl ${CURL_IP_OPTS} http://localhost:${coord_port}/lra-recovery-coordinator/recovery
     # sometimes it can take two scans to complete recovery
-    curl http://localhost:${coord_port}/lra-recovery-coordinator/recovery 
+    curl http://localhost:${coord_port}/lra-recovery-coordinator/recovery
```
Comment (mayankkunwar) [inline rts/lra-examples/run.sh:82]: Is IPV4 not required here?

Comment (mmusgrov) [reply rts/lra-examples/run.sh:82]: This curl is sending an HTTP GET request to a JAX-RS endpoint which will be running inside a container that accepts IPv4 requests.

Comment (mmusgrov) [reply rts/lra-examples/run.sh:82]: +1, the other PR changes all use ${CURL_IP_OPTS} in the curl request.

Comment (ochaloup) [reply rts/lra-examples/run.sh]: thanks, I overlooked this

**Comment by ochaloup** [comment]
I added a next JBTM issue commit to fix the LRA example to be processed on CI: https://issues.redhat.com/browse/JBTM-3282

**Comment by ochaloup** [comment]
ok, finally the PR passed...
@mayankkunwar would you be so kind and review this PR and aprove/decline for me being able to process (merge/fix). Thanks.

**Pull request merged**